AUTHORS

```diff
@@ -1,17 +1,17 @@
-# List of authors for Stockfish
+# List of Stockfish authors
 
 # Founders of the Stockfish project and fishtest infrastructure
-Tord Romstad (romstad)
-Marco Costalba (mcostalba)
 Joona Kiiski (zamar)
+Tord Romstad (romstad)
 Gary Linscott (glinscott)
+Marco Costalba (mcostalba)
 
-# Authors and inventors of NNUE, training, NNUE port
+# Authors and inventors of NNUE, training, and NNUE port
 Yu Nasu (ynasu87)
```

Comment:
the authors here are not alphabetical

Reply:
Yeah, you're right. I have now alphabetically ordered everything.

Reply:
no, I meant these authors are in a specific order, and there is no need to change that.